kite-data/kite-data-core/src/test/java/org/kitesdk/data/event/thrift/Value.java

```diff
@@ -0,0 +1,427 @@
+/**
+ * Autogenerated by Thrift Compiler (0.7.0)
```

Comment:
Ugh, I didn't know the maven plugin requires the executable installed. That's annoying.